packages/oas/src/operation/index.ts

```diff
@@ -708,7 +708,9 @@ export class Operation {
    *
    */
   getRequestBodyExamples(): RequestBodyExamples {
-    if (this.requestBodyExamples) {
+    const nonEmptyRequestExample = this.requestBodyExamples?.[0]?.examples?.[0].value;
```

Comment:
this is ugly :(

Reply:
lol nauuur it's great! would this also solve the problem at hand? roughly the same from an ugliness standpoint but i think it's safer since it would allow falsy values like `0`, `false`, etc. (might also be good to have a test case for falsy values too!)

```suggestion
    const isRequestExampleValueDefined = typeof this.requestBodyExamples?.[0]?.examples?.[0].value !== 'undefined';
```